### PR TITLE
Pull request for issue #2676 to include episode aired date in newsletter

### PR DESCRIPTION
**Pull request by AdamWorley**
I have only implemented the aired date for newly added to plex as that's what was requested and I'm unfamiliar with emby.
issue #2676

### CODE REVIEW

File: src/Ombi.Api.Plex/Models/Metadata.cs [hunk 1]
```diff
@@ -19,7 +19,7 @@ public class Metadata
         public string banner { get; set; }
         public string theme { get; set; }
         //public string duration { get; set; }
-        //public string originallyAvailableAt { get; set; }
```
Comment (tidusjar) [inline src/Ombi.Api.Plex/Models/Metadata.cs:22]: Can we please undo this change?

File: src/Ombi.Helpers/StringHelper.cs [hunk 2]
```diff
@@ -77,6 +77,57 @@ public static int IntParseLinq(string stringIn)
             return -1;
         }
 
+        public static string BuildEpisodeList(IEnumerable<(int EpisodeNumber, DateTime? Aired)> orderedEpisodes)
```
Comment (tidusjar) [inline src/Ombi.Helpers/StringHelper.cs:80]: Looks good, but don't feel like this belongs in the `StringHelper`? It's very specific to what's going on

**Review by tidusjar — CHANGES_REQUESTED** [review]
We are now also adding the `Aired` property onto each of the entities, there needs to be data migrations created for SQLite and MySQL for this. Then they should also be populated when they are created (Not just in the RecentlyAddedEngine, e.g. the PlexContentSync). 

So far you have done a good job! Just it's not totally finished.

The other option is instead of storing the AirDate against the actual entity, in the NewsLetterJob we actually look up the media on TheMovieDb/TvMaze which should also contain that information, so we could easily leverage that

**Comment by AdamWorley** [comment]
I have made the changes you (@tidusjar) have recommended and I have leveraged the Movie db for the season/episode info.

I opted to use the tv/season endpoint, instead of the episode endpoint to avoid spamming api requests.

**Pull request closed without merging**